runner/organizations.go

```diff
@@ -63,11 +60,20 @@ func (r *Runner) CreateOrganization(ctx context.Context, param params.CreateOrgP
 		}
 	}()
 
-	poolMgr, err := r.loadOrgPoolManager(org)
+	internalCfg, err := r.getInternalConfig(org.CredentialsName)
```

Comment:
Calling ```getInternalConfig()``` and setting the ```Internal``` field every time before calling ```CreateOrgPoolManager()``` feels a bit tedious. The internal config is useful to the pool manager, but doesn't make sense to have it be a part of the ```Organization{}``` or ```Repository{}```. I made a mistake adding it there. If we're refactoring this bit anyway, and creating an interface for controllers, I think we should separate that info from the repo and org structs.

That information can be part of the ```PoolManager{}``` implementation or part of the ```PoolManagerController{}``` implementation. The information in the internal config is from the config file, which can't really be updated without a restart, so it makes sense to initialize it once, inject it into the pool manager controller.

This would allow us to clean up this bit of repetitive code. What do you think?

Reply:
I agree that the code with internal config is a bit repetitive. 
I will try to clean this up.